go.mod

```diff
@@ -11,10 +11,10 @@ require (
 	github.com/filecoin-project/go-amt-ipld/v3 v3.1.1 // indirect
```

Comment:
Update the version of go used by the project, its line 3 of this file

Reply:
Fixed!